DigitalPassaport/DigitalPassaport/Infactuture/Resources/NetworkManager.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  NetworkRoutable.swift
+//  DigitalPassaport
+//
+//  Created by Milena Predic on 29.8.23..
+//
+import Foundation
+import Combine
+import Alamofire
+
+class NetworkManager {
```

Comment:
We don't need this class. This configuration should be part of NetworkService, and this router should be independent for each service, that corresponds to each repository.

Reply:
Changed implementation